ddp-workspace/projects/ddp-sdk/src/lib/services/serviceAgents/serviceAgent.service.ts

```diff
@@ -69,7 +69,8 @@ export class ServiceAgent<TEntity> {
         path: string,
         body: any,
         options: any = {},
-        throwErrorObject: boolean = false): Observable<any> {
+        throwErrorObject: boolean = false,
+        logErrorToCloud: boolean = true): Observable<any> {
```

Comment:
New param `logErrorToCloud` to determine whether to send failed request to GCP stackdriver logging.
If `/profile/address/verify` request fail, do not log it as ERROR.